tests/test_adapters.py

```diff
@@ -88,15 +88,30 @@ async def test_adapters_are_cached():
     assert len(adapter_store.adapters) == 1
     # Same unique ID is reused for the second request
     assert (
-        adapters_1["lora_request"].lora_int_id == adapters_2["lora_request"].lora_int_id
+        adapters_1["prompt_adapter_request"].prompt_adapter_id == adapters_2["prompt_adapter_request"].prompt_adapter_id
+    )
+
+@pytest.mark.asyncio
+async def test_lora_adapters_are_not_cached():
+    adapter_name = "granite-3b-code-instruct-lora"
+    request = BatchedGenerationRequest(
+        adapter_id=adapter_name,
     )
 
+    adapter_store = AdapterStore(cache_path=FIXTURES_DIR, adapters={})
+
+    adapters_1 = await validate_adapters(request, adapter_store=adapter_store)
+    adapters_2 = await validate_adapters(request, adapter_store=adapter_store)
+
+    # Metadata isn't cached in our cache
+    assert len(adapter_store.adapters) == 0
```

Comment:
Yes, though we do want to also ensure it's cached in the REST server's cache.

I don't have a high "care level" of how that's implemented- you could have `adapter_store.adapters` add the data from the REST cache so that these tests don't actually change, and instead we ensure that e.g.
```
assert adapter_store.adapters[adapter_name] == vllm_server.state.openai_serving_models.lora_requests[0]
```

Or, keep this as is and don't cache here and simply
```
assert vllm_server.state.openai_serving_models.lora_requests[0] == adapters_1
```